src/StoreApi/Utilities/ProductQuery.php

```diff
@@ -31,17 +31,36 @@ public function prepare_objects_query( $request ) {
 			'paged'               => $request['page'],
 			'post__in'            => $request['include'],
 			'post__not_in'        => $request['exclude'],
-			'posts_per_page'      => $request['per_page'],
+			'posts_per_page'      => $request['per_page'] ? $request['per_page'] : -1,
```

Comment:
I know that if `per_page` is set to 0, this will evaluate to -1, but why does that? wouldn't make sense to let the client use the same logic as WP?

Reply:
it's easier to check ! per_page then set to -1 than handle 0 and -1 separately. The schema validates from 0 and has a message than leaving blank, or using 0, maps to unlimited.

Some functions in WP use 0 instead of -1 - I think the terms query object.